schedule.yaml

```diff
@@ -186,9 +182,13 @@ tabs:
     description: null
     time: 12:00 - 13:00
     label: all
+  - title: Project Updates
+    description: null
+    time: 13:00 - 14:00
```

Comment:
Do we need an hour for this?

Reply:
No, we can actually remove this from the schedule.

Reply:
I am not actually sure what this item on the schedule is. I'll check with Charley to see if this is something she planned, or if it can be removed altogether.

Reply:
Talked to Steven and we can remove this
@JessicaS11

Reply:
@spestana Are we removing for all days or just Thursday?

Reply:
Just Thursday